Makefile

```diff
@@ -27,16 +27,24 @@ clean: ## Removes build and test artifacts
 	@find . -name '*~' -exec rm -f {} +
 	@find . -name '__pycache__' -exec rm -rf {} +
 
-.PHONY: docker-build
-docker-build:
-	@echo "==> Building docker image for local testing"
-	@docker build -t dag_factory:latest .
+
+.PHONY: copy-files
+copy-files: ## copy project in docker context
+	mkdir -p dev/include/tmp_dagfactory
+	cp -r ./dagfactory dev/tmp_dagfactory
+	cp ./pyproject.toml dev/include/tmp_dagfactory/pyproject.toml
+	cp ./LICENSE dev/include/tmp_dagfactory/LICENSE
+	cp ./README.md dev/include/tmp_dagfactory/README.md
+	cp -r ./examples/* dev/dags/
```

Comment:
Why don't we just move examples to `dev/dags`?

Reply:
We could move the example in dags files, but maybe we need to adjust paths in some tests/readme, etc. Please do let me know if we want to do this changes in this PR?

Reply:
We agreed to have a symbolic link from `examples` and keep the data in `dev/dags`

Reply:
I created a symbolic link between examples/* and dev/dags but DAG is not visible in Airflow 

**Symbolic link created locally**

<img width="1153" alt="Screenshot 2024-10-21 at 6 52 05 PM" src="https://github.com/user-attachments/assets/1bb08cd5-e192-40fb-b393-ea9887d3a10a">

**Symbolic link visible in docker**

<img width="1449" alt="Screenshot 2024-10-21 at 6 53 21 PM" src="https://github.com/user-attachments/assets/96ae13d7-d0ef-4201-8665-dd2f7413e121">

**DAG is not visible in Airflow UI**

<img width="1703" alt="Screenshot 2024-10-21 at 6 53 32 PM" src="https://github.com/user-attachments/assets/c0ecc76a-2a32-4b72-848d-cc8650b503ee">

Reply:
The files are in the examples folder. Are you suggesting moving them to dev/dags, or am I missing something?

Reply:
Yes, @pankajastro, that's the suggestion
`git mv examples dev/dags`

And then create a symbolic link `examples`